src/app/startups/applications/page.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+import { database } from "@/app/firebaseConfig";
+import PaginationSection from "@/components/pagination-section";
+import SearchCard from "@/components/search-card";
+import TableSectionAplications from "@/components/table-section-applications";
+import Title from "@/components/title";
+import { ref, onValue } from "firebase/database";
+import { useState, useEffect } from "react";
+
+const ITEMS_PAGE = 6; // Mostrar 6 elementos por página
```

Comment:
@CristhianRF24 please comments in English

Reply:
It looks a chatGPT code 
I have no problem with it but please , read the code not just copy and paste